Source/Extensions/Blazorise.Components/Autocomplete.razor

```diff
@@ -49,50 +50,47 @@
 
         @if ( DropdownVisible )
         {
-            for ( int i = 0; i < FilteredData.Count; ++i )
+            @if ( Virtualize )
             {
-                var item = FilteredData[i];
-                var isActiveItem = IsSuggestedActiveItem( item );
-                var isFocusedItem = i == ActiveItemIndex;
+                <Virtualize TItem="TItem" Context="item" Items="@FilteredData">
+                    @itemFragment(item)
+                </Virtualize>
```

Comment:
**ReadData** support by implementing the appropriate handler?

Reply:
You have more experience with this. Do you want to try it?

Reply:
Tested with **ReadData** before starting to work on this and works already really good already.

But if there's huge ammounts of data, the ItemsProvider should be more performant as you'll only be querying that data specifically (similar to paging), but for rendering purposes it's already taking the advantage of the **Virtualization**.

Reply:
Done, will you proceed with the rest of the PR? :)

Reply:
I guess I will :)